index.html

```diff
@@ -1,279 +1,292 @@
+                  <div class="grid-item"> <span class="canop">FACEBOOK</span></div>
+                  <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> Full Stack Dev</div>
+                  <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> 2015</div>
+               </div>
+               <div class="doc">
+                  <p>
+                    Experimental content creation feature that allows users to add
+                     to an existing story over the course of a day without spamming their friends.                    
+                  </p>
+               </div><br>
+               <div class="tags">
+                  <ul class="works">
+                     <li> <a href="#" class="btn"> html </a></li>
+                     <li> <a href="#" class="btn"> css </a></li>
+                     <li> <a href="#" class="btn"> javascript </a></li>
+                     <li> <a href="#" class="btn" id="rub"> Ruby On Rails</a></li>
+                  </ul>
+               </div>
+               <a href="#">
+                  <div class="btn-main">See Project</div>
+               </a>
             </section>
-        </article >
-    
-        <article  class="snapshot1">
+         </article >
+         <!--Snap 4  -->
+         <article  class="snapshot1">
             <img src="img/Snapshoot Portfolio4.png" alt="snapshot1" class="image-po">
-            <section id="main">            
-                <div class="head-one-alt">
-                    <h2>Multi-Post <br>
-                        Stories</h2>
-                </div>
-                    <div class="grid-container">
-                        <div class="grid-item"> <span class="canop">CANOPY</span></div>
-                        <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> Back End Dev</div>
-                        <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> 2015</div>  
-                    </div>               
-                
-                <div class="doc">
-                    <p>
-                        A daily selection of privately personalized reads; no accounts or sign-ups required.                    
-                    </p>
-                </div>
-                <div class="tags">
-                    <ul class="works">
-                        <li> <a href="#" class="btn"> html </a></li>
-                        <li> <a href="#" class="btn"> css </a></li>
-                        <li> <a href="#" class="btn"> javascript </a></li>
-                    </ul>
-                </div>
-                <a href="#"> <div class="btn-main">See Project</div> </a> 
-                
+            <section class="main" id="snap-item">
+               <div class="head-one-alt">
+                  <h2> Uber Navigation
+                  </h2>
+               </div>
```

Comment:
Looks great, but please make sure to follow the exact Figma design, and some padding to the left.

![image](https://user-images.githubusercontent.com/34779928/194699767-79477484-e005-4021-b5e0-feb86f3ba90b.png)

Reply:
Fixed.